### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#146] [Esercizi - Bertini] [answered]
![Es 6](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/61630471-d7e7-4abd-9a4e-7b2a73877ee0)

**Comment by daromacarlo** [comment]
![immagine](https://github.com/user-attachments/assets/f94a4970-9346-4b52-bf1f-baf2f063bd36)

**Comment by LorenzoBurello** [comment]
Non sono d'accordo con l'esercizio 2. Così non stai calcolando la probabilità che ti esca colore con un solo seme?

**Comment by daromacarlo** [comment]
@LorenzoBurello vero, moltiplico per 4

**Comment by CiottoloMaggico** [comment, marked as answer]
# Soluzione confermata in classe
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/84e6e343-92fd-4a80-a161-8c8cee5a4978)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/dce64f49-1c44-447d-9d43-094be6945564)
@CarloDaRomadev nel punto 4 conti l'ordine in cui le 2 coppie di carte dello stesso valore si trovano nel mazzo, nel punto 5 lo stesso

**Comment by daromacarlo** [comment]
@CiottoloMaggico 4) prendo 4 carte, tra queste ne scelgo 2 e moltiplico le combinazioni per i 13 valori. Prendo altre 4 carte, tra queste ne scelgo 2 e moltiplico per i 12 valori rimasti, infine scelgo l'ultima carta tra le carte rimaste.

Non è giusto come ragionamento?

**Comment by CiottoloMaggico** [comment]
Immagina di avere una mano doppia coppia, [ Asso (C), Asso (P), 7 (C), 7 (P), 3 (Q) ]
facendo 13*(4 scelgo 2) * 12(4 scelgo 2) * 44 conti:
la mano [Asso (C), Asso (P), 7 (C), 7 (P), 3 (Q) ] e la mano  [7 (C), 7 (P), Asso (C), Asso (P), 3 (Q) ]  quando in realtà sono la stessa mano.

Al contrario con (13 scelgo 2)(4 scelgo 2)^2*44, se ci sono prima i 7 o gli assi non importa (A, A, 7, 7) = (7, 7, A, A).
Se ci fai caso, svolgendo i calcoli la mia formula conta esattamente la metà delle combinazioni contate dalla tua proprio per questo motivo.

**Comment by daromacarlo** [comment]
@CiottoloMaggico aaah aaaaah okok, il 13*12, grazie

**Comment by CiottoloMaggico** [comment]
@LorenzoBurello il 16 = 4² = ( 4 scelgo 1 ) ( 4 scelgo 1 ) = i possibili modi di scegliere il seme della prima carta e quello della seconda

**Comment by Elia-Belli** [comment]
Numeriamo le carte francesi da 1 a 52: 
$\Omega = \\{(w_1,..,w_5)| w_1,..,w_5=1,..,52,w_i\ distinte\\}, |\Omega|= \binom{52}{5}= \frac{52!}{5!(47)!}=62375040$

- $A = \\{poker\\}$
 Estraggo la prima carta (mi importa il valore), altre 3 dovranno avere lo stesso valore e l'ultima può essere qualsiasi delle rimanenti: $|A|= 13 \cdot 48$
 $\mathbb{P}(A)= 624/\binom{52}{5}$
- $B = \\{colore\\}$ (incluso scala reale)
 Estraggo la prima carta (mi importa il seme) e le altre dovranno avere lo stesso seme: $|B|= 4 \cdot \binom{13}{5}$
$\mathbb{P}(B)= 4 \cdot \binom{13}{5}/\binom{52}{5}$
- $C = \\{full\\}$
Estraggo la prima carta e ne scelgo un'altra con lo stesso valore, estraggo la terza carta con valore diverso dalla prima e ne estraggo altre due con lo stesso valore: $|C|=(13 \cdot \binom{4}{2})\cdot (12 \cdot \binom{4}{3})$
$\mathbb{P}(C)=|C|/\binom{52}{5}$
- $D = \\{doppia\ coppia\ ma\ non\ full\\}$ (escludendo i poker)
 Estraggo la prima carta e ne estraggo un'altra con lo stesso valore, estraggo la terza carta di valore diverso dalla prima e ne estraggo un'altra con lo stesso valore, estraggo l'ultima di valore diverso dalle altre: $|D|=\binom{13}{2} \cdot \binom{4}{2}\cdot \binom{4}{2} \cdot (11\cdot 4)$
$\mathbb{P}(D)=|D|/\binom{52}{5}$
- $E = \\{tris\ ma\ ne'\ poker\ ne'\ full\\}$
 Estraggo la prima carta e ne estraggo altre 2 dello stesso valore, scelgo la quarta di valore diverso dalla prima, scelgo la quinta di valore diverso dalla prima e dalla quarta: $|E|=(13 \cdot \binom{4}{3})\cdot \binom{48}{2}-|C|$
$\mathbb{P}(E) = |E|/\binom{52}{5}$
